src/content/docs/tips/tailwindv4.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Tailwind V4
+description: Quick set up guide for Tailwind v4
+---
+import { Tabs, TabItem, LinkCard } from "@astrojs/starlight/components";
+
+How to use Tailwind's v4 Vite plugin instead of Astro's Tailwind integration.
+
+Be sure to check out the [Tailwind v4 Blogpost](https://tailwindcss.com/blog/tailwindcss-v4-alpha#using-vite) for details on using v4.
+
+## Install
+
+<Tabs>
+    <TabItem label="npm">
+        ```sh
+        npm install --save-dev @tailwindcss/vite
+        ```
+    </TabItem>
+    <TabItem label="pnpm">
+        ```sh
+        pnpm add -D @tailwindcss/vite
+        ```
+    </TabItem>
+    <TabItem label="yarn">
+        ```sh
+        yarn add --dev @tailwindcss/vite
+        ```
+    </TabItem>
+</Tabs>
+
+## Configure
+
+Next, update your [Prettier config](https://prettier.io/docs/en/configuration) in the project's root to include `prettier-plugin-astro` plugin. This example will use `.prettierrc.json`:
+
+```ts title="astro.config.mjs"
+import { defineConfig } from 'astro/config';
+import tailwindcss from '@tailwindcss/vite';
+
+// https://astro.build/config
+export default defineConfig({
+  vite: {
+    plugins: [tailwindcss()],
+    css: {
+      transformer: 'lightningcss'
```

Comment:
Does it require a specific vite version?

Reply:
I don't think so but good question @alexanderniebuhr do you know?

Reply:
Yeah it does need `>4.4.0` _(https://github.com/vitejs/vite/blob/v5.2.9/packages/vite/CHANGELOG.md#440-2023-07-06)_

Reply:
ok how does this work... I think since `2.10` astro ships with  `^>4.4.0` but does that mean it's compatible with astro versions where vite is at least version `>4.0.0` and if thats the case is it work mentioning?